pkg/render/apiserver_test.go

```diff
@@ -1111,7 +1111,7 @@ var _ = Describe("API server rendering tests (Calico Enterprise)", func() {
 			// - 1 added by the operator by default
 			// - 1 added by the calicoNodeDaemonSet override
 			Expect(d.Spec.Template.Annotations).To(HaveLen(2))
-			Expect(d.Spec.Template.Annotations).To(HaveKey("hash.operator.tigera.io/tigera-apiserver-certs"))
+			Expect(d.Spec.Template.Annotations).To(HaveKey("tigera-operator.hash.operator.tigera.io/tigera-apiserver-certs"))
```

Comment:
I see that in certificatebundle L114 we have the following: `annotations[fmt.Sprintf("%s.hash.operator.tigera.io/%s", cert.GetNamespace(), cert.GetName())] = hash`
And that leads to this value on this line.

I was wondering if a better key for the hash would be: `hash.operator.tigera.io/%s.%s"`